xen/drivers/passthrough/arm/ipmmu-vmsa.c

```diff
@@ -1169,7 +1168,7 @@ static int ipmmu_attach_device(struct iommu_domain *io_domain,
 		return ret;
 
 	for (i = 0; i < archdata->num_utlbs; ++i)
```

Comment:
BTW, don't we have coding style issue here, e.g. need a space after brace: for (i -> for ( i